File: SoundSesh/Portal/SoundSesh.Angular/ClientApp/src/app/shared/models/feedback.ts
import { DateTime } from 'luxon';

export class Feedback {
    public workitem: string;
    public importance: string;
    public willingtopayforfeature: string;
    public willingtopayforfeatureamount: string;
    public willingtopayforsubscription: string;
    public willingtopayforsubscriptionamount: string;
    public comments: string;
    public createdate: DateTime;
    public userid: string;
}

File: SoundSesh/Studios/SoundSesh.Studios.Entities/Models/Feedback.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace SoundSesh.Studios.Entities.Models
{
    [Table("Feedback")]
    public class Feedback : Trackable
    {
        public string WorkItem { get; set; }
        public string Importance { get; set; }
        public string WillingToPayForFeature { get; set; }
        public string WillingToPayForFeatureAmount { get; set; }
        public string WillingToPayForSubscription { get; set; }
        public string WillingToPayForSubscriptionAmount { get; set; }
        public string Comments { get; set; }
        public string UserId { get; set; }
    }
}